IGCexpansion/CodonGeneconv.py

```diff
@@ -406,16 +429,23 @@ def get_MG94Geneconv_and_MG94(self):
                 Qb = Qbasic[sb, sa]
                 if isNonsynonymous(cb, ca, self.codon_table):
                     Tgeneconv = self.get_IGC_nonsynonymous_contribution()
+                    if self.use_Homo_Omega():
+                        rate_geneconv.append(Qbasic_Homo[sb, sa])

```

Comment:
`+ Tgeneconv`?

Reply:
Move all `if self.use_Homo_Omega()` to where `Qb = xxx` so that `Qb is set by these if else conditions`